roboteam_ai/src/control/ControlGoToPos.cpp

```diff
@@ -101,11 +98,14 @@ void ControlGoToPos::goToPosBasic(RobotPtr robot, Vector2 &targetPos) {
         pidBasic.setD(0);
         far = false;
     }
+    //if (dist > rtt::ai::constants::ROBOT_RADIUS) {
+    //    std::cout << "Robot:" << robot->id << "TargetPos:" << dist << std::endl;
+    //}
     Vector2 delta = pidBasic.controlPIR(error, robot->vel);
     Command command;
     command.id = robot->id;
     command.use_angle = 1;
-    command.w = static_cast<float>(delta.angle());
+    command.w = 0;
```

Comment:
why did you change this functionality?

Reply:
because they spin more with 0, the angle changes every time, and so I felt it would interfere with possible debugging

Reply:
So now they always look at the same way? We want our robots to face the direction they are going.

Reply:
yeah, but let's fix that when we fix the constant turning, inputting a changing angle might also make them turn constantly, so I would like to keep those issues separate.

Reply:
Still: Why change it here then? Make a separate branch for this and fix the entire problem with turning there at once. We do not want partial solutions on development, because then people might think this was the intended functionality.

Reply:
yeah, but wasn't the problem somewhere in grsim? This is the least bug-prone/least-intended-side-effects version